src/azul/service/manifest_service.py

```diff
@@ -1394,8 +1397,15 @@ def command_lines(cls,
         file_options = [
             '--fail-early',  # Exit curl with error on the first failure encountered
             '--continue-at -',  # Resume partially downloaded files
-            '--retry 2',  # Retry a file download up to X times on transient error
-            '--retry-delay 10',  # Sleep for X seconds between retries
+            # We want curl to make enough retries so that it waits a total of
+            # one and a half times the evaluation period of the WAF rate rule.
+            # This lets us be sure that if curl tripped the rate rule, it will
+            # wait long enough for the tripped rule to clear.
```

Comment:
```suggestion
            # We want curl to make enough retries so that it waits a total of
            # one and a half times the evaluation period of the WAF rate rule, long enough for the tripped rule to clear.
```